public/app/plugins/panel/barchart/module.tsx

```diff
@@ -78,7 +78,7 @@ export const plugin = new PanelPlugin<BarChartOptions, BarChartFieldConfig>(BarC
       })
       .addSliderInput({
         path: 'valueRotation',
```

Comment:
should we also change the real property value?  `labelRotation`?

Reply:
Yeah, think this should be changed

Reply:
changed to xTickLabels[Rotation/MaxLength]